### DISCUSSION TITLE
Validator for URL encoded format

### DISCUSSION THREAD

**Discussion by ssathishan** [typestack/class-validator#2042] [Questions & Answers]
Does this component has validator to validate URL encoded string? 

_Example:_
**Input:** %3c%73%63%72%69%70%74%3e%61%6c%65%72%74%28%36%37%38%29%3c%2f%73%63%72%69%70%74%3e

Above is the encoded string of '<script>alert(678)</script>'

I want to validate this input .

**Comment by VardyNg** [comment]
Would be a great feature indeed
